tests/messages/test_messaging_1x1_chat.py

```diff
@@ -128,14 +140,18 @@ def test_1x1_chat(multiple_instances):
             message = chat.find_message_by_text(chat_message_reply, 0)
             assert message.reply_corner.exists
 
+        with step(f'User {user_one.name}, add reaction to the last message and verify it was added'):
+            message.open_context_menu_for_message().add_reaction_to_message(1)
+            assert driver.waitFor(lambda: HEART_EMOJI_PATH == message.get_emoji_reactions_pathes()[0], timeout)
+
```

Comment:
@Valentina1133 is it possible to select random reaction each time (we have i think only 5 of them) when adding it to a message? That will add a bit more randomness to our tests which is good

Reply:
Yes, but in this case I don’t know how to check if correct emoji was added. The only option I found how to verify it - checking image source path, this will be difficult to verify in case of adding of random reaction

Reply:
I have one idea actually, will try it

Reply:
maybe we can do it like this?
- select a random emoji and click it (any of the items from imagePath list, as we certainly know which ones we use, it should be known from source code)
- hover the message and check which emoji was applied and store the value
- and then check it on the recipient side?

ya it kind of requires extra step for sure

Reply:
@Valentina1133 , got some inspiration from Lukasz:

```
    function getShortcodeFromId(emojiId) {
        switch (emojiId) {
            case 1: return ":heart:"
            case 2: return ":thumbsup:"
            case 3: return ":thumbsdown:"
            case 4: return ":laughing:"
            case 5: return ":cry:"
            case 6: return ":angry:"
            default: return undefined
        }
    }
```

and this is the corresponding mapping to filenames:
```
    proc setup(self: Model) =
        self.items = @[
            initItem(1, "emojiReactions/heart",      false),
            initItem(2, "emojiReactions/thumbsUp",   false),
            initItem(3, "emojiReactions/thumbsDown", false),
            initItem(4, "emojiReactions/laughing",   false),
            initItem(5, "emojiReactions/sad",        false),
            initItem(6, "emojiReactions/angry",      false),
        ]
```

Reply:
I added random selection of emoji reaction and verifications both on sender and recipient sides that correct emoji added.